crates/aptos/src/node/local_testnet/logging.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright © Aptos Foundation
```

Comment:
All of this isn't strictly necessary, we could just write all our tracing output to a single file, but separating it by service makes it easier to understand.

Reply:
Why not use the already existing logging framework?

Reply:
The txn stream services don't use the existing logger bc we didn't want to bring in all those deps, so it uses tracing instead.

Besides that, so do you know if aptos_logger supports writing to different files on a per Tokio worker basis?